linera-views/src/backends/rocks_db.rs

```diff
@@ -86,56 +76,26 @@ impl RocksDbStoreInternal {
 
     fn build(
         path_with_guard: PathWithGuard,
-        namespace: &str,
         max_stream_queries: usize,
         cache_size: usize,
         root_key: &[u8],
     ) -> Result<RocksDbStoreInternal, RocksDbStoreError> {
-        let mut full_path_buf = path_with_guard.path_buf.clone();
-        full_path_buf.push(root_key_as_string(root_key));
+        let full_path_buf = path_with_guard.path_buf.clone();
```

Comment:
`full_path_buf` -> `path` or `db_path`. Not sure the `.clone()` is indeed.

Reply:
Ok, for the renaming.
The `.clone()` is needed.